node/src/traits.rs

```diff
@@ -90,6 +90,7 @@ pub trait NodeInterface<N: Network>: Routing<N> {
         tokio::task::spawn(async move {
             match signal_listener().await {
                 Ok(()) => {
+                    warn!("Orderly shutdown started, please wait until it is complete.");
```

Comment:
This message needs to made more urgent and clear about the potential consequences of early termination.

Also these two messages should be made into an alert color like yellow or orange and bolded if possible.

```suggestion
                    warn!("Graceful node shutdown started!");
                    warn!("NOTE: PLEASE WAIT UNTIL THE SHUTDOWN PROCESS COMPLETES TO AVOID DATABASE CORRUPTION");
```

Reply:
Note: an unclean shutdown shouldn't cause any db corruption (since we write atomically), and if it does, it implies a bug. I haven't experienced it in a long time regardless of how the shutdown looked like.

Reply:
The point of this PR to ensure people avoid sending `SIGKILL` or initiating any other kind of non-graceful shutdown  and leaving the node in an inconsistent state. Thus it should be emphasized very clearly what the potential consequences are.

In theory we write atomically but until there's recorded evidence from tests that repeated non-graceful shutdowns does not corrupt data, we should not be making the assumption that it will never do so.

Reply:
Subsequent signals are ignored as the signal handler is just sleeping and then exiting.

Reply:
We can not bold the text of the initial message as we don't know at that point if we are running in TTY mode.